### PR TITLE
Changed pronouns

### PR DESCRIPTION
**Pull request by techieeliot**
## Description

Fixed a broken link from the Hacktoberfest blog post and searched out for ways to update the documentation, specifically pronouns.

## Related Issues
n/a

### CODE REVIEW

File: packages/gatsby-plugin-sharp/README.md [hunk 1]
```diff
@@ -213,7 +212,7 @@ the original image, allowing the original image and its colors to partially
 "shine through". _Heads up_: If the original image contains an alpha channel it
 will be [flattened][15] before creating the composite.
 
-This works by adding an alpha channel to the duotone'd image - then we let Sharp
+This works by adding an alpha channel to the duotone'd image - then you let Sharp
```
Comment (laurieontech) [inline packages/gatsby-plugin-sharp/README.md]: This is something the plugin does, so we would be appropriate.
```suggestion
This works by adding an alpha channel to the duotone'd image - then we let Sharp
```

**Comment by laurieontech** [comment]
Thanks for the PR! Since these are two unrelated files could you split this PR? One for the broken link and another for the README changes? That helps us review them faster.

**Comment by techieeliot** [comment]
Ok, I'm learning how to split the PR in git and will update.

Starting with this link on [How to Split Pull Requsts](https://www.thedroidsonroids.com/blog/splitting-pull-request). 

Is the `git cherry-pick` approach the best way to create a new branch and PR?

File: packages/gatsby-plugin-sharp/README.md (hunk 1 above)
Comment (marcysutton) [reply packages/gatsby-plugin-sharp/README.md]: I'd recommend using "then the plugin lets Sharp do its magic"

File: packages/gatsby-plugin-sharp/README.md [hunk 2]
```diff
@@ -24,8 +24,7 @@ of PNGs then it can significantly reduce build times.
 
 ## How to use
 
-```javascript
-// In your gatsby-config.js
```
Comment (marcysutton) [inline packages/gatsby-plugin-sharp/README.md]: This only applies to README files: GitHub doesn't render the title thing, so I'd keep this in here. It's one of the only cases where we can be more specific for GitHub (unlike linking, which we keep relative to gatsbyjs.org).

File: packages/gatsby-plugin-sharp/README.md (hunk 1 above)
Comment (techieeliot) [reply packages/gatsby-plugin-sharp/README.md]: Thanks @laurieontech and @marcysutton. I will use Marcy's suggestion if that's cool?

File: packages/gatsby-plugin-sharp/README.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ of PNGs then it can significantly reduce build times.
 
 ## How to use
 
-```javascript
+```javascript	```javascript:title=gatsby-config.js
```
Comment (marcysutton) [inline packages/gatsby-plugin-sharp/README.md]:
````suggestion
```javascript:title=gatsby-config.js
````

**Comment by marcysutton** [comment]
This PR is small enough, I'd say it's probably not worth the effort to split it. If there were 10+ files and a ton of different changes, that would apply for sure!

File: packages/gatsby-plugin-sharp/README.md [hunk 4]
```diff
@@ -24,19 +24,20 @@ of PNGs then it can significantly reduce build times.
...
 plugins: [
-  {
-    resolve: `gatsby-plugin-sharp`,
-    options: {
-      useMozJpeg: false,
-      stripMetadata: true,
-      defaultQuality: 75,
-    },
-  },
+{
```
Comment (wardpeet) [inline packages/gatsby-plugin-sharp/README.md]: could you put back the indentation we had before?

File: packages/gatsby-plugin-sharp/README.md (hunk 1 above)
Comment (wardpeet) [reply packages/gatsby-plugin-sharp/README.md]: Really cool! 😎

File: packages/gatsby-plugin-sharp/README.md [hunk 5]
```diff
@@ -165,7 +166,7 @@ allImageSharp {
     }
   }
 }
-```
+````
```
Comment (wardpeet) [inline packages/gatsby-plugin-sharp/README.md]:
````suggestion
```
````

Was this a mistake?

**Comment by LekoArts** [comment]
@techieeliot Hi! 👋 Do you have time to address the requested changes?

**Comment by LekoArts** [comment]
Hey! Thanks so much for opening a pull request!

The change you’ve proposed is not going to be accepted because this change doesn't meaningfully improve docs, fix a bug, or otherwise improve functionality.

We absolutely want to have you as a contributor, so please take a look at [our open issues][open-issues] for ideas, and please reach out to us on Twitter at `@gatsbyjs` with questions.

We offer pair programming sessions if you’d like to work with one of our maintainers to make your first contribution.

Thanks again, and we look forward to seeing more PRs from you in the future! :muscle::purple_heart:

[open-issues]: https://github.com/gatsbyjs/gatsby/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc

**Comment by techieeliot** [comment]
No problem @LekoArts. Sorry, I had set this aside and forgotten to fix the PR by splitting it up. I would be interested in pairing with one of the maintainers, so I can contribute. 

Thank you.

**Pull request closed without merging**